management/models.py

```diff
@@ -43,6 +43,8 @@ class Contact(models.Model):
     org_sign_up = models.CharField(max_length=40, blank=True)
     hospital_name = models.CharField(max_length=50, blank=True)
     doctor_name = models.CharField(max_length=30, blank=True)
+    preg_signup = models.NullBooleanField(default=False)
+    preg_update = models.NullBooleanField(default=False)
```

Comment:
We should document all these fields, I don't think I really understand what they are or what they do.